swagger-apis/enrollments/1.0.0-beta.1.yml

```diff
@@ -832,29 +836,29 @@ components:
                   type: object
                   properties:
                     latitude:
-                      type: number
-                      example:
+                      type: number 
+                      example: #adicionar exemplo, definir maximum
                     longitude:
-                      type: number
-                      example:
+                      type: number #definir format, definir maximum
```

Comment:
não seria melhor ser do tipo string e com um pattern para definir o formato?

Reply:
É possível mas não parece trazer algum ganho real. Não temos como garantir a quantidade de casas decimais, se forçar minLength ainda sim seria possível preencher tudo com zero.
Parece um erro conceitual usar regex pra validar intervalos que são numéricos por definição: de -90 +90 e  de -180 a + 180.

Reply:
Opa, entendi sua consideração, Nic, mas o format não é pattern, serve só pra deixar claro se o tipo "Number" é no formato float ou double. Não utilizamos regex em format, só em pattern.